README.md

```diff
@@ -1,9 +1,1444 @@
+action_result.data.\*.BackupDetails.BackupArn | string |  `aws dynamodb backup arn`  |   arn:aws:dynamodb:us-east-1:157568067690:table/cars/backup/01663158543903-413ed1de 
+action_result.data.\*.BackupDetails.BackupCreationDateTime | string |  |   2022-09-14 12:29:03.903000+00:00 
+action_result.data.\*.BackupDetails.BackupExpiryDateTime | string |  |   2022-09-22 11:30:03.118000+00:00 
+action_result.data.\*.BackupDetails.BackupName | string |  |   cars_backup 
+action_result.data.\*.BackupDetails.BackupSizeBytes | numeric |  |   266 
+action_result.data.\*.BackupDetails.BackupStatus | string |  |   CREATING 
+action_result.data.\*.BackupDetails.BackupType | string |  `aws dynamodb backup type`  |   USER 
+action_result.data.\*.ResponseMetadata.HTTPHeaders.connection | string |  |   keep-alive 
+action_result.data.\*.ResponseMetadata.HTTPHeaders.content-length | string |  |   252 
+action_result.data.\*.ResponseMetadata.HTTPHeaders.content-type | string |  |   application/x-amz-json-1.0 
+action_result.data.\*.ResponseMetadata.HTTPHeaders.date | string |  |   Wed, 14 Sep 2022 12:29:03 GMT 
+action_result.data.\*.ResponseMetadata.HTTPHeaders.server | string |  |   Server 
+action_result.data.\*.ResponseMetadata.HTTPHeaders.x-amz-crc32 | string |  |   1789868498 
+action_result.data.\*.ResponseMetadata.HTTPHeaders.x-amzn-requestid | string |  |   K109PP9CK2RGFK6BV1HDE87C5JVV4KQNSO5AEMVJF66Q9ASUAAJG 
+action_result.data.\*.ResponseMetadata.HTTPStatusCode | numeric |  |   200 
+action_result.data.\*.ResponseMetadata.RequestId | string |  |   K109PP9CK2RGFK6BV1HDE87C5JVV4KQNSO5AEMVJF66Q9ASUAAJG 
+action_result.data.\*.ResponseMetadata.RetryAttempts | numeric |  |   0 
+action_result.summary | string |  |  
+action_result.message | string |  |   Created backup successfully 
+summary.total_objects | numeric |  |   1 
+summary.total_objects_successful | numeric |  |   1   
+
+## action: 'restore backup table'
+Create a new table from an existing backup
+
+Type: **generic**  
+Read only: **False**
+
+<p> <strong>Restore Secondary Indexes</strong> <p> Restore Secondary Indexes parameter is used to restore the define the secondary indexes of the new restored table. There are following two options for the parameter: <ul> <li><strong>Restore the entire table</strong> <p>Restore table with original indexes as they are in the backup</p> </li> <li> <strong> Restore the table without secondary indexes </strong> (Selected by Default) <p> It Restores table without any indexes. There will be no Local and Global Secondary Index. Incase you want to add indexes, you can only create Global Secondary Indexes. Local Secondary Indexes cannot be created once the table is created. </p> </li> </ul> </p> </p> <p> <strong>NOTE</strong> <p> If while restoring table <strong>Restore the table without secondary indexes</strong> option is selected then there will be no Local Secondary Index. </p> </p>
```

Comment:
The first sentence is confusing and doesn't make sense. I'm not too familiar with this app but I feel like the first sentence should be "The Restore Secondary Indexes parameter is used to restore the secondary indexes of the new restored table."

Reply:
Updated the code and resolved the issue.